src/main/java/gui/DashboardWindow.java

```diff
@@ -134,108 +127,108 @@ public DashboardWindow() {
-                    System.out.println(customPhilosophy);
-
-                    DecisionCostEvaluator decisionCostEvaluator =
-                            new DecisionCostEvaluator(consequenceContainer, factory, customPhilosophy);
-
-                    for (Decision decision : collidedEntities.keySet()) {
-                        decisionCosts.put(getActionNameFromDecision(decision.toString()), decisionCostEvaluator.getSummarizedCostForDecision(decision));
-                    }
-
-                    String bestDecision = BusinessLogic.getOptimumDecision(decisionCosts);
-
-                    jLabelBestDecision.setText("Best decision: " + bestDecision);
-                    jLabelBestDecision.setVisible(true);
-
-                    int numberOfDecisions = decisionCosts.size();
-
-
-                    String[] columnNames = {"Action", "Moral cost"};
-                    Object[][] data = new Object[numberOfDecisions][2];
-                    int i = 0;
-                    for(String decisionName : decisionCosts.keySet()){
-                        data[i][0] = decisionName;
-                        data[i][1] = decisionCosts.get(decisionName);
-                        i++;
-                    }
-                    DefaultTableModel defaultTableModel = new DefaultTableModel(data, columnNames) {
-                        private static final long serialVersionUID = 1L;
-                    };
-                    JTable jTable = new JTable(defaultTableModel);
-
-                    jScrollPaneWithResults = new JScrollPane(jTable);
-                    int jScrollPanelHeight = (jTable.getRowCount()+1)*jTable.getRowHeight() + 7;
-                    if(jScrollPanelHeight > 150) jScrollPanelHeight = 150;
-                    jScrollPaneWithResults.setBounds(CENTER_CUSTOM_PHILOSOPHIES + 220, 600, 400, jScrollPanelHeight);
-                    add(jScrollPaneWithResults);
-                    System.out.println("eleoe");
+
+                String[] columnNames = {"Action", "Moral cost"};
+                Object[][] data = new Object[numberOfDecisions][2];
+                int i = 0;
+                for (String decisionName : decisionCosts.keySet()) {
+                    data[i][0] = changeSnakeCase(decisionName);
+                    data[i][1] = decisionCosts.get(decisionName);
+                    i++;
                 }
+                DefaultTableModel defaultTableModel = new DefaultTableModel(data, columnNames) {
+                    private static final long serialVersionUID = 1L;
+                };
+                JTable jTable = new JTable(defaultTableModel);
+
+                jScrollPaneWithResults = new JScrollPane(jTable);
+                int jScrollPanelHeight = (jTable.getRowCount() + 1) * jTable.getRowHeight() + 7;
+                if (jScrollPanelHeight > 150) jScrollPanelHeight = 150;
+                jScrollPaneWithResults.setBounds(CENTER_CUSTOM_PHILOSOPHIES + 220, 600, 400, jScrollPanelHeight);
```

Comment:
w całym tym pliku jest sporo magicznych stałych podanych przez wartość, nie wiem czy da się to jakoś wyciągnąć i czy to będzie ładne? Łatwiej by się edytowało później na pewno

Reply:
Ogólnie to jest masa funkcji setBounds wywoływanych na komponentach i wszystkie oprócz tej są obok siebie, argumenty to 4 inty.  Może byłoby ładnie, ale wyglądałoby to tak, że było 10 albo więcej zmiennych odpowiedzialnych za te wymiary i niekoniecznie byłoby to czytelne